File: 8월 1주차/breadbirds/03/[PGS - Lv1] 부족한 금액 계산하기.java
class Solution {
    public long solution(int price, int money, int count) {
        long answer = -1;
        // 이용료 price
        // 몇 번 탔는지 count
        // 내 돈 money
        // N번 탔을 때 이용료 total
        long total = 0;
        for(int i = 1; i <= count; i ++){
            total += price *i;
        }
        if(total <= money){
            answer = 0;
        }
        else answer = total - money;
        

        return answer;
    }
}

File: 2026 08월/8월 4주차/JeongEon8/01/[PGS - Lv1] 부족한 금액 계산하기.cpp
#include <cmath>

using namespace std;

long long solution(int price, int money, int count)
{
    long long answer = -1;
    long long sum = 0;    
    
    for(int i = 1; i <= count; i++){
        sum += price * i;
    }
    
    if(money > sum){
        answer = 0;
    }
    else{
        answer = abs(money - sum);        
    }

    return answer;
}
